generate-status/collect_info.py

```diff
@@ -0,0 +1,35 @@
+from kubernetes import client, config
+import gitlab
+
+g = gitlab.Gitlab(context["gitlab_url"] , private_token=context["gitlab_private_key"])
+gitlab_group = g.groups.get(context["gitlab_group"])
+
+config.load_incluster_config()
+custom_object_api = client.CustomObjectsApi()
+anarchy_subject_list = custom_object_api.list_namespaced_custom_object(
+  group="anarchy.gpte.redhat.com",
+  version="v1",
+  namespace=context["babylon_namespace"],
+  plural="anarchysubjects",
+)
+
+def getAnarchySubject(project_id):
+  subject = list(filter(lambda x: x["metadata"]["annotations"]["poolboy.gpte.redhat.com/resource-claim-name"] == f"labs.ocp4.na-{project_id}", anarchy_subject_list["items"]))
```

Comment:
I'm wondering if we can make this more generic? The `labs.ocp4.na` string will change as we move into other regions (and when we support more than just OCP4).
Do you think it would be enough to just check for `{project_id}` in the value of `poolboy.gpte.redhat.com/resource-claim-name`?